### PR TITLE
Add link to branch in build header

### PR DESCRIPTION
**Pull request by lislis**
screen shot 2016-11-15 at 17 16 47

### CODE REVIEW

**Comment by joshk** [comment]
👍 from me!

File: app/components/build-header.js [hunk 1]
```diff
@@ -42,5 +42,9 @@ export default Ember.Component.extend({
 
   elapsedTime: Ember.computed('item.startedAt', 'item.finishedAt', 'item.duration', function () {
     return durationFrom(this.get('item.startedAt'), this.get('item.finishedAt'));
+  }),
+
+  urlGitHubBranch: Ember.computed('item.repo.slug', 'item.branch.name', function () {
+    return githubBranch(this.get('item.repo.slug'), this.get('item.branch.name'));
   })
```
Comment (backspace) [inline app/components/build-header.js]: This is perfect, but I thought you might be interested to see a peek of the future, available as the [`ember-computed-decorators`](https://github.com/rwjblue/ember-computed-decorators) addon, which would change this with the proposed ES2017 (maybe?) decorators feature. If you haven’t already heard:

```
@computed('item.repo.slug', 'item.branch.name')
urlGitHubBranch(slug, branchName) {
  return gitHubBranch(slug, branchName);
}
```

I think it’ll be nice to not have that duplication of specifying the dependent keys in the computed property header and then having to fetch them within the function.

File: app/templates/components/build-header.hbs [hunk 2]
```diff
@@ -37,6 +37,12 @@
           {{/if}}
         </li>
       {{/if}}
+      <li>
+        <a class="commit-branch-url" href="{{urlGitHubBranch}}" title="Look at the branch on GitHub">
+          {{inline-svg 'images/stroke-icons/icon-github' class="icon"}}
+          <span class="label-align">Branch {{item.branch.name}}</span>
+        </a>
+      </li>
```
Comment (backspace) [inline app/templates/components/build-header.hbs]: I seem to have found a bug:

image

While looking at this via PR deployment, it was working. Then I clicked the branch name to view it on GitHub, pressed back, and the branch name was missing. 😢

The link ends with `undefined` too, sadly.

(This isn’t really a comment on the template code, I just don’t think I can make a general comment without submitting the entire review.)

File: app/components/build-header.js (hunk 1 above)
Comment (clekstro) [inline app/components/build-header.js]: Just wanted to mention I'm a huge fan of that addon and I'd love to see us move in that direction. Decorators are currently stage 2, so it'll still probably be awhile until they're implemented natively, but I think the benefits are high enough for us to justify the investment.

File: app/templates/components/build-header.hbs (hunk 2 above)
Comment (lislis) [inline app/templates/components/build-header.hbs]: hmm weird. I can't reproduce this neither locally nor on the PR deploy 😕

File: app/components/build-header.js (hunk 1 above)
Comment (lislis) [inline app/components/build-header.js]: I haven't heard about this yet! thanks for sharing

File: app/templates/components/build-header.hbs (hunk 2 above)
Comment (lislis) [inline app/templates/components/build-header.hbs]: After some more back and forth clicking I was able to reproduce it once. I'll try and see if I can find something

**Pull request merged**